src/Widgets/EligibilityModal/modal.test.tsx

```diff
@@ -86,6 +86,23 @@ describe('Modal', () => {
       )
     })
   })
+  describe('deffered plan provided', () => {
+    it('should display the correct title', () => {
+      render(
+        <EligibilityModal
+          eligibilityPlans={mockPlansAllEligible}
+          initialPlanIndex={1}
+          status={apiStatus.SUCCESS}
+          onClose={() => {
+            console.log('modal closed')
+          }}
+        />,
+      )
+      expect(screen.getByTestId('modal-title-element')).toHaveTextContent(
+        'Payez en plusieurs fois ou plus tard par carte bancaire avec Alma.',
+      )
+    })
+  })
```

Comment:
A good starting point but the test should cover both cases:
- one where the `currentPlan` value is a deferred payment plan
- one where it's not
If you want to do it elegantly, you can use this type of syntax: https://jestjs.io/docs/api#2-describeeachtablename-fn-timeout

Reply:
perfect 👍